### PR TITLE
LPS-129589 Migrate asset tag selection modals

### PR DESCRIPTION
**Pull request by markocikos**
Hey @liferay-echo ,

JIRA ticket: https://issues.liferay.com/browse/LPS-129589

Forward from https://github.com/liferay-frontend/liferay-portal/pull/978 . This is the second of three asset PRs, that have been on hold for a while, waiting for GA release. First one was https://github.com/liferay-echo/liferay-portal/pull/4436

Notes:
- In the original PR, I cleaned up `eventName` from the tag, as the standard multiple selection in a search container does not require it. But, I reverted those changes in this PR. This keeps changes simple and local, and also allows easier enabling of different types of selection if you ever decide to do it. For example, we have a single selection in this PR. It's not done through a tag, but through direct referencing of a jsp.

Test case 1, single target tag. The only use is in segments:
- People > Segmens > (+) > drag & drop 'User / Tag' > Select (see gif)

after1

Test case 2, multiselect. There a lot of places where we have this, as a part of `<AssetTagsSelector>`. Here is one example:
- Asset Publisher > Configuration > Asset Selection - Dynamic > Filter > Select a Tag (see gif)

after2

/cc @kresimir-coko @jonmak08

### CODE REVIEW

File: modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js [hunk 1]
```diff
@@ -132,44 +131,42 @@ function AssetTagsSelector({
...
+						value: item.value,
+					};
+				});
+
+				const addedItems = newValues.filter(
+					(newValue) =>
+						!selectedItems.find(
+							(selectedItem) =>
+								selectedItem.label === newValue.label
+						)
```
Comment (kresimir-coko) [inline modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js:150]: This looks weird 😕 

Does it negate the found `selectedItem`? Why the `!`?

Maybe I'm missing something, didn't have my coffee yet.

**Comment by victorg1991** [comment]
Hey @markocikos! Segments belongs to the tango team, would you mind sending this pr to https://github.com/liferay-tango ? :)

Thanks!!

**Comment by victorg1991** [comment]
@markocikos realised that the second test case belong to us, let me test that real quick and you can forward it to verify the other one!

**Comment by victorg1991** [comment]
Nevermind the code belong to us anyway 😂 forget all my comments

**Comment by victorg1991** [comment]
sorry for the noise!

**Comment by victorg1991** [comment]
This Looks good to me :) Let's see if the ci agrees

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**